server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileService.java

```diff
@@ -40,41 +40,18 @@
   private final RuleActivator ruleActivator;
   private final QProfileFactory factory;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "factory" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1068)